components.json

```
@@ -910,6 +910,10 @@
 			"require": "clike",
 			"owner": "just-boris"
 		},
+		"psl": {
+			"title": "PSL",
```

Another thing: The title.

When googling "PSL" I could only find the [Property Specification Language](https://en.wikipedia.org/wiki/Property_Specification_Language), so let's not use the abbreviation.

```suggestion
			"title": "PATROL Scripting Language",
```